scripts/index.js

```diff
@@ -14,24 +12,30 @@ movieResultsElement.innerHTML = `
 //No searches found message
 function renderNoResultsMessage() {
     movieResultsElement.innerHTML = ` 
-    <div class="no-search-placeholder"> 
-    <i class="fa-solid fa-film"></i> <p>No results found, sorry!</p>
-</div>
+        <div class="no-search-placeholder"> 
+            <i class="fa-solid fa-film"></i>
+            <p>No results found, sorry!</p>
+        </div>
 `
 }
 
 //Loading message
 function renderLoadingMessage() {
     movieResultsElement.innerHTML = `
-    <div class="loading-placeholder">
-    <i class="fa-solid fa-spinner"></i> <p> Result are loading... </p>
-    </div>
-    `
+        <div class="loading-placeholder">
+            <i class="fa-solid fa-spinner"></i>
+            <p> Result are loading... </p>
+        </div>
+`
 }
 
 function searchMovieDatabase() {
     renderLoadingMessage()
-    fetch(`https://www.omdbapi.com/?apikey=863d41df&s=${searchInput.value}&type=movie`)
+    let requestUrl = `https://www.omdbapi.com/?apikey=863d41df&type=movie`
+    if (searchInput.value) {
+        requestUrl = `https://www.omdbapi.com/?apikey=863d41df&s=${searchInput.value}&type=movie`
+    }
+    fetch(requestUrl)
```

Comment:
This makes it so we don't make a with an empty `search` parameter.

If we've typed something into the search box, we'll include that in the request, otherwise we won't bother with it.